addons/editor-theme3/theme3.js

```diff
@@ -879,6 +885,8 @@ export default async function ({ addon, console, msg }) {
         )
       );
       workspace.refreshTheme();
+      // used by editor-colored-context-menus
+      document.body.style.setProperty("--colour-text", uncoloredTextColor());
```

Comment:
This seems unnecessary - the variable is only referenced by colored context menus in a rule targeting `.blocklyMenuItem`, which only exists in new Blockly.

Reply:
It's inside the `if (Blockly.registry)` block. I didn't see an existing `theme3` CSS variable that I could use but I can set `--sa-contextmenu-text` and use that on both editor versions if you prefer.

Reply:
Oops, I read that wrong. Setting `--colour-text` is fine, but you could also use `--sa-block-gray-text`.